lib/tasks/send_reminder_email.rake

```diff
@@ -0,0 +1,21 @@
+namespace :send_reminder_email do
+    desc 'Sends a reminder email to user to address reminder type'
+    task days: :environment do
+        puts Date.today
+        User.all.each do |user|
+            user.subscriptions.all.each do |subscription|
+                subscription.reminders.all.each do |reminder|
+                    #for each reminder, check if the end date and calculate date to send reminder
+                    endDate = reminder.end_date
+                    reminderDate = endDate - reminder.time_delta
+                    puts reminderDate
+                    if reminderDate == Date.today
+                        puts "Reminder date today"
+                        ReminderMailer.with(user: user, reminder: reminder).new_reminder_email.deliver_now
```

Comment:
are the reminders deleted after they are sent? (if not, can we add that?)

Reply:
Currently no, it depends on if they are recurring (Iteration 5) - but I could add that. If Ritchey has a preference on this, we can go with that. Otherwise, we can talk about how we go about deleting reminders.